packages/fxa-payments-server/src/components/PaymentForm/index.tsx

```diff
@@ -203,7 +203,7 @@ export const PaymentForm = ({
             let error = null;
             value = ('' + value).substr(0, 5);
             if (!value) {
-              error = 'Zip code is required';
```

Comment:
Same here - without validating on empty zip field, the submit button goes active as long as only the CC-related fields are filled and the checkbox is checked